<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Copyright 2016, Christian Kvasny</author>
 <class>QRKGastroManagerTableEdit</class>
 <widget class="QDialog" name="QRKGastroManagerTableEdit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>227</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tische Editieren</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="nameLabel">
       <property name="text">
        <string>Tisch Name:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="name">
       <property name="placeholderText">
        <string>Tischname</string>
       </property>
       <property name="clearButtonEnabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="categorieLabel">
       <property name="text">
        <string>Raum</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="groupComboBox"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Button Farbe:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QComboBox" name="colorComboBox"/>
     </item>
     <item row="4" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_2"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QrkPushButton" name="okButton">
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>60</height>
        </size>
       </property>
       <property name="text">
        <string>OK</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>:src/icons/ok.png</normaloff>:src/icons/ok.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="autoDefault">
        <bool>true</bool>
       </property>
       <property name="default">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QrkPushButton" name="cancelButton">
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>60</height>
        </size>
       </property>
       <property name="text">
        <string>Abbrechen</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>:src/icons/cancel.png</normaloff>:src/icons/cancel.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="autoDefault">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QrkPushButton</class>
   <extends>QPushButton</extends>
   <header>qrkpushbutton.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>name</tabstop>
  <tabstop>groupComboBox</tabstop>
  <tabstop>colorComboBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
